diff --git a/src/main/java/com/floragunn/searchguard/dlic/rest/validation/TenantValidator.java b/src/main/java/com/floragunn/searchguard/dlic/rest/validation/TenantValidator.java
index 40c27c1d..ff8e7aa0 100644
--- a/src/main/java/com/floragunn/searchguard/dlic/rest/validation/TenantValidator.java
+++ b/src/main/java/com/floragunn/searchguard/dlic/rest/validation/TenantValidator.java
@@ -24,3 +24,3 @@ public class TenantValidator extends AbstractConfigurationValidator {
 		super(request, ref, esSettings, param);
-		this.payloadMandatory = false;
+		this.payloadMandatory = true;
 	    allowedKeys.put("description", DataType.STRING);
